resources/tests/kueue/test-preemption.yml

```
@@ -0,0 +1,146 @@
+name: test-kueue-job
+description: submit and validate a kueue job
+tasks:
+- id: register-cluster-queue
+  type: RegisterObj
+  params:
+    template: "resources/templates/kueue/cluster-queue.yml"
+- id: register-local-queue
+  type: RegisterObj
+  params:
+    template: "resources/templates/kueue/local-queue.yml"
+- id: register-resource-flavor
+  type: RegisterObj
+  params:
+    template: "resources/templates/kueue/resource-flavor.yml"
+- id: register-job
+  type: RegisterObj
+  params:
+    template: "resources/templates/kueue/job.yml"
+    nameFormat: "job{{._ENUM_}}"
+    podNameFormat: "{{._NAME_}}-[0-9]-.*"
+    podCount: "{{.parallelism}}"
+- id: create-resource-flavor
+  type: SubmitObj
+  params:
+    refTaskId: register-resource-flavor
+    params:
+      name: "gpu-node"
+      nodeLabel:
+        name: "nvidia.com/gpu.count"
+        value: "8"
```

Should this be:

```suggestion
        "nvidia.com/gpu.count": "8"
```

https://kueue.sigs.k8s.io/docs/concepts/resource_flavor/

---

oh, right, thank you